node/external/nodeApiResolver.go

```diff
@@ -339,12 +339,18 @@ func (nar *nodeApiResolver) GetManagedKeysCount() int {
 	return nar.managedPeersMonitor.GetManagedKeysCount()
 }
 
-// GetManagedKeys returns all keys managed by the current node when running in multikey mode
+// GetManagedKeys returns all keys that should act as validator(main or backup that took over) and will be managed by this node
 func (nar *nodeApiResolver) GetManagedKeys() []string {
 	managedKeys := nar.managedPeersMonitor.GetManagedKeys()
 	return nar.parseKeys(managedKeys)
 }
 
+// GetLoadedKeys returns all keys that were loaded and will be managed by this node
+func (nar *nodeApiResolver) GetLoadedKeys() []string {
+	loadedKeys := nar.managedPeersMonitor.GetLoadedKeys()
```

Comment:
Question: should we append the main BLS key from crypto components to this list? Reason: maybe we are using the node in single key mode, not multikey and this route should return the main BLS key. 
Or return that single key if the list returned by the managedPeersMonitor is empty. I guess this will be better as the multikey list won't be polluted by an observer BLS key.

Reply:
pushed

it will also return the public key for single key observers